lib/src/api_client.dart

```diff
@@ -43,6 +50,33 @@ class ApiClient {
     params['adsetId'] = adsetId;
     if (amount != null) params['amount'] = amount;
 
+    return handlePostRequest(params);
+  }
+
+  Future<void> callEvent({
+    required String sessionId,
+    required String deviceId,
+    required String storeId,
+    required String path,
+    String? customerId,
+    String? productCode,
+    String? title,
+    String? referrer,
+  }) async {
+    final params = <String, dynamic>{};
+    params['sessionId'] = sessionId;
+    params['deviceId'] = deviceId;
+    params['storeId'] = storeId;
+    params['path'] = path;
+    if (customerId != null) params['customerId'] = customerId;
+    if (productCode != null) params['productCode'] = productCode;
+    if (title != null) params['title'] = title;
+    if (referrer != null) params['referrer'] = referrer;
+
+    return handlePostRequest(params);
+  }
+
+  Future<void> handlePostRequest(Map<String, dynamic> params) async {
```

Comment:
로컬에서만 사용하는 함수는 무조건 언더바를 붙힙니당
(핸들링하는걸 상속받는 클래스 마다 커스텀을 원했다면 public도 맞는것같은데
아직은! public하게 조정하는게 없으니깐 private function이 맞는 것 같아용)
```suggestion
  Future<void> _handlePostRequest(Map<String, dynamic> params) async {
```

Reply:
아하 넵 감사합니다!